improver_tests/calibration/quantile_regression_random_forests_calibration/test_load_and_train_quantile_regression_random_forest.py

```diff
@@ -69,6 +94,7 @@ def add_wind_data_to_forecasts(data_dict, wind_speed_values, wind_dir_values):
     wind_dir_dict["units"] = "degrees"
     wind_dir_dict["diagnostic"] = "wind_direction_at_10m"
     wind_dir_dict["experiment"] = "recentblend"
+    wind_dir_dict[representation] = np.nan
```

Comment:
This is a notable change in this file, as previously the wind direction representation was not overwritten, so was the same as the wind speed. If the wind direction provided is actually deterministic then the representation value will be NaN.